docs/docsource/Reference/bucket_website_operations/bucket_website_specification.rst

```diff
@@ -3,7 +3,7 @@
 Bucket Website Specification
 ============================
 
-Scality Zenko Enterprise implements the `AWS S3 Bucket Website
+Scality Zenko implements the `AWS S3 Bucket Website
```

Comment:
Scality Zenko or just Zenko? My preference is for `Zenko` only

Reply:
I agree, this looks stupid. This type of use is going to require manual intervention. Flag this and use it to build another ticket. I'll see what I can do after hours.